hebrew/bezMan/testing.txt

```diff
@@ -0,0 +1 @@
+check this!
```

Comment:
thanks! is it possible to ask questions here?

2016-08-09 5:39 GMT+03:00 Casey Collaborator <notifications@github.com>:

> In hebrew/bezMan/testing.txt
> <https://github.com/udacity/create-your-own-adventure/pull/5207#discussion_r73990311>
> :
>
> > @@ -0,0 +1 @@
> > +check this!
>
> It looks like this page is not reachable from the rest of the story. That
> is, if a reader starts in language.md and keeps clicking links, they will
> never find this page. You can fix this by finding a good place in the
> existing story to add an option with a link that leads to this page.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/5207/files/422a5a9781350d309dce27dc35bba1ae987d5779#r73990311>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFELd75VuI6a7HzlNG29oBINnnO2ts9kks5qd-hTgaJpZM4Je0nc>
> .
>